src/test/java/org/opensearch/security/auditlog/impl/DisabledCategoriesTest.java

```diff
@@ -195,7 +193,7 @@ protected boolean categoriesPresentInLog(String result, AuditCategory... categor
 	}
 
 	protected void logAll(AuditLog auditLog) {
-		//11 requests
+		//10 requests
```

Comment:
What does this comment mean?

Reply:
The comment was in there before, and I think it just states the obvious ;-) namely, we log/generate 10 events in this method. Agreed, it's unnecessary and I can remove it if needed.

Reply:
If you have an opportunity to remove lets do it, or we filter it out over time